src/model/TaskState.model.js

```diff
@@ -0,0 +1,23 @@
+import { Enum } from 'enumify'
+
+/**
+ * Cylc valid task states.
+ */
+class TaskState extends Enum {}
+TaskState.initEnum([
+  'RUNAHEAD',
+  'WAITING',
+  'HELD',
+  'QUEUED',
+  'READY',
```

Comment:
These will need update soon, with recent changes and discussions.

Reply:
Agreed, I was going to send this as a separate PR, but thought it was better to push the component as-is instead of trying to perfect it too much.

Reply:
(later I will update the rest of the code to use this enum)